### ISSUE TITLE
Trigger on label event

### ISSUE THREAD

**Issue by CiaranOMara** [JuliaImages/ImageContrastAdjustment.jl#34]
I'm reading through various benchmark workflows to get some ideas as to how to set up my own.

I think the comment below may no longer be true.
https://github.com/JuliaImages/ImageContrastAdjustment.jl/blob/5642b3a02e4a24c7f2b528f64870e56f7ab3393e/benchmark/run_benchmarks.jl#L6-L7

Combining:
- https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif
- https://help.github.com/en/actions/reference/events-that-trigger-workflows#label-event-label
- https://developer.github.com/webhooks/event-payloads/#label

I think a label trigger can be achieved with the following YAML adjustments.
```yaml
on:
  label:
    types: [created, edited]

...

steps:
 - name: Triggered by label
   if: ${{ github.event_name == 'label' && github.event.label.name == 'run benchmark' }}
```

The `label` event will only trigger a workflow run if the workflow file is on the master or default branch.

All totally untested of course.

**Comment by zygmuntszpak** [comment]
Thank you very much for the suggestion. I haven't had time to really look into the benchmark stuff (I think Johnny set it up), but when I do I will check to see if I can incorporate your advice.

**Comment by johnnychen94** [comment]
Closed by #37 

@CiaranOMara The on-label events seem not to work as expected, see #35 and #36. Or did I miss something?

**Comment by CiaranOMara** [comment]
@johnnychen94, you've pushed me over the line to test my suggestion. I found that the on label event gets triggered when creating, editing, or deleting a label, which is not like the activity of labelling a pull request or issue. I think you captured that distinction with your solution - nice!
